packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -35,177 +52,383 @@ const MyProfile = observer(() => {
-                                                    max_characters={300}
-                                                />
-                                            )}
-                                        </Field>
-                                        <Field name='default_advert_description'>
-                                            {({ field }) => (
-                                                <Input
-                                                    {...field}
-                                                    type='textarea'
-                                                    label={localize('Instructions')}
-                                                    error={errors.default_advert_description}
-                                                    hint={localize('This information will be visible to everyone.')}
-                                                    className='my-profile__form-textarea'
-                                                    has_character_counter
-                                                    initial_character_count={
-                                                        my_profile_store.default_advert_description.length
-                                                    }
-                                                    max_characters={300}
-                                                />
-                                            )}
-                                        </Field>
-                                        <div className='my-profile__footer'>
-                                            <FormError message={my_profile_store.form_error} />
+                            </div>
+                            <div className='my-profile__separator'>
+                                <div className='my-profile__separator-text'>{localize('Privacy setting')}</div>
+                                <div className='my-profile__separator-horizontal_line' />
+                            </div>
+                            <div className='my-profile__privacy-setting'>
+                                <ToggleSwitch
+                                    id='p2p_profile_real_name'
+                                    classNameLabel='my-profile__privacy-setting-switch'
+                                    classNameButton='my-profile__privacy-setting-switch-button'
+                                    handleToggle={() => {
+                                        toggleShowRealName(!show_real_name);
+                                    }}
+                                    is_enabled={show_real_name}
+                                />
+                                <span className='my-profile__privacy-setting-text'>
+                                    {localize('Show my real name')}({general_store.nickname})
+                                </span>
+                            </div>
```

Comment:
This is a part of a different card which involves API, etc. This is the card: https://trello.com/c/2NhbICmq/139-carol-p2p-allow-user-to-use-real-name.